controllers/packagevariants/pkg/controllers/packagevariant/packagevariant_controller.go

```diff
@@ -334,6 +350,13 @@ func (r *PackageVariantReconciler) ensurePackageVariant(ctx context.Context,
 	if err = r.Client.Create(ctx, newPR); err != nil {
 		return nil, err
 	}
+
```

Comment:
Looks good. I'd be interested to see how this works in a full test-infra e2e run.
Also, may be somewhat aligned with what @kispaljr was working on?